### ISSUE TITLE
[1.20.1] liquid blaze burner texture missing

### ISSUE THREAD

**Issue by KLONOA9X6** [mrh0/createaddition#623]
**Describe the bug**
![image](https://github.com/mrh0/createaddition/assets/95112579/e32e54ae-0b2a-45e5-88d7-f28244c51bf5)

**To Reproduce**
Use a straw to blaze burner.

**Additional context**
- Mod version: createaddition-1.20.1-1.0.0
- Forge version: 47.1.43
- Create version: 1.20.1-0.5.1.d

**Comment by KLONOA9X6** [comment]
Mods:
```
ad_astra-forge-1.20.1-1.15.1.jar
appleskin-forge-mc1.20.1-2.5.0.jar
architectury-9.0.8-forge.jar
AttributeFix-Forge-1.20.1-21.0.1.jar
balm-forge-1.20-7.0.7.jar
BenefitsofReading-Forge-1.20.X-1.0.0.jar
BetterModsButton-v8.0.2-1.20.1-Forge.jar
BetterPingDisplay-1.20.1-1.1.jar
BetterPvP_23.5.0_Forge_1.20.jar
BetterThirdPerson-Forge-1.20-1.9.0.jar
botarium-forge-1.20-2.1.0.jar
canary-mc1.20.1-0.2.7.jar
cfm-forge-1.20.1-7.0.0-pre36.jar
cloth-config-11.0.99-forge.jar
craftgr-1.4.8-mc1.20-forge.jar
create-1.20.1-0.5.1.d.jar
createaddition-1.20.1-1.0.0.jar
createdieselgenerators-1.20.1-1.1c.jar
createoreexcavation-1.20-1.3.0.jar
createsifter-1.20.1-1.6.1.d-9.jar
create_enchantment_industry-1.20.1-for-create-0.5.1.d-1.2.6.jar
create_jetpack-forge-4.0.1.jar
create_mechanical_extruder-1.20.1-1.5.1.d-9.jar
cuisinedelight-1.1.8.jar
curios-forge-5.2.0-beta.3+1.20.1.jar
dummmmmmy-1.20-1.7.9.jar
dynamiclightsreforged-1.20.1_v1.5.0.jar
fancymenu_forge_2.14.9_MC_1.20.1.jar
FarmersDelight-1.20.1-1.2.3.jar
ferritecore-6.0.0-forge.jar
Forgematica-0.1.0-mc1.20.1.jar
geckolib-forge-1.20.1-4.2.1.jar
HutoolCore-1.2.0-Hutool-5.8.16-For-Forge-Fabric-Paper-Ignite.jar
IBEEditor-1.20-2.2.3-forge.jar
IMBlocker_4.0.5+1.19.4.jar
InventoryProfilesNext-forge-1.20-1.10.6.jar
Jade-1.20.1-forge-11.4.3.jar
JadeAddons-1.20.1-forge-5.1.0.jar
konkrete_forge_1.6.1-2_MC_1.20.jar
kotlinforforge-4.3.0-all.jar
l2artifacts-2.4.8.jar
l2complements-2.4.14.jar
l2library-2.4.15.jar
l2weaponry-2.4.8.jar
libIPN-forge-1.20-3.0.2.jar
MaFgLib-0.1.2-mc1.20.1.jar
MaxHealthFix-Forge-1.20.1-12.0.1.jar
mcw-bridges-2.1.1-mc1.20.1forge.jar
mcw-doors-1.1.0forge-mc1.20.1.jar
mcw-fences-1.0.7-mc1.20.1forge.jar
mcw-lights-1.0.6-mc1.20.1forge.jar
mcw-paths-1.0.3forge-mc1.20.1.jar
mcw-roofs-2.2.4-mc1.20.1forge.jar
mcw-trapdoors-1.1.1-mc1.20.1forge.jar
memoryleakfix-forge-1.17+-1.1.1.jar
ModernUI-1.20.1-3.7.1.3-universal.jar
moonlight-1.20-2.6.0-forge.jar
netherportalfix-forge-1.20-13.0.0.jar
oculus-flywheel-compat-1.20.1-0.2.0.jar
oculus-mc1.20-1.6.4.jar
REIPluginCompatibilities-forge-12.0.55.jar
resourcefulconfig-forge-1.20-2.0.0.jar
resourcefullib-forge-1.20.1-2.1.1.jar
RoughlyEnoughItems-12.0.634.jar
rubidium-0.6.5.jar
rubidium_extras-1.20.1_v1.4.3.jar
supplementaries-1.20-2.5.18.jar
Terralith_1.20.1_v2.4.3.jar
TextruesRubidiumOptions-1.0.0+mc1.20.jar
toms_storage-1.20-1.6.4.jar
TravelersBackpack-1.20.1-9.1.4.jar
Tweakerge-0.1.0-mc1.20.1.jar
twilightdelight-2.0.2.jar
twilightforest-1.20.1-4.3.1740-universal.jar
voicechat-forge-1.20.1-2.4.16.jar
XaerosWorldMap_1.31.0_Forge_1.20.jar
yesstevemodel-1.20-release-1.1.5-hotfix-2.jar
```

**Comment by mrh0** [comment]
fixed in 1.0.0b.
